tests/system-tests/rdscore/internal/rdscorecommon/egress-ip.go

```diff
@@ -0,0 +1,1018 @@
+	"context"
+	"fmt"
+	"math/rand"
+	"net"
+	"net/netip"
+	"strings"
+	"time"
+
+	"github.com/openshift-kni/eco-goinfra/pkg/bmc"
+	"github.com/openshift-kni/eco-goinfra/pkg/nodes"
+	"github.com/openshift-kni/eco-gotests/tests/system-tests/rdscore/internal/rdscoreparams"
+
+	"github.com/openshift-kni/eco-goinfra/pkg/namespace"
+
+	"k8s.io/apimachinery/pkg/util/wait"
+
+	"github.com/openshift-kni/eco-goinfra/pkg/pod"
+	scc "github.com/openshift-kni/eco-gotests/tests/system-tests/internal/scc"
+
+	"github.com/openshift-kni/eco-gotests/tests/system-tests/internal/apiobjectshelper"
+
+	"github.com/golang/glog"
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+	"github.com/openshift-kni/eco-goinfra/pkg/clients"
+	"github.com/openshift-kni/eco-goinfra/pkg/deployment"
+	"github.com/openshift-kni/eco-goinfra/pkg/egressip"
+	. "github.com/openshift-kni/eco-gotests/tests/system-tests/rdscore/internal/rdscoreinittools"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+const (
+	proberRBACRole       = "system:openshift:scc:privileged"
+	nonEgressIPPodLabel  = "env=centos"
+	nonEgressIPNamespace = "non-egressip-ns"
+)
+
+var (
+	egressIPPodLabelsMap = map[string]string{
+		strings.Split(RDSCoreConfig.EgressIPPodLabel, "=")[0]: strings.Split(RDSCoreConfig.EgressIPPodLabel, "=")[1]}
+
+	egressIPPodSelector = metav1.ListOptions{
+		LabelSelector: RDSCoreConfig.EgressIPPodLabel,
+	}
+	nonEgressIPPodSelector = metav1.ListOptions{
+		LabelSelector: nonEgressIPPodLabel,
+	}
+)
+
+// createAgnhostRBAC creates the SCC privileged, serviceAccount and RBAC.
+func createAgnhostRBAC(
+	apiClient *clients.Settings,
+	egressIPNamespace string) (string, error) {
+	proberDeploySAName := fmt.Sprintf("rdscore-prober-sa-%s", egressIPNamespace)
+	proberDeployRBACName := fmt.Sprintf("privileged-rdscore-rbac-%s", egressIPNamespace)
+
+	var err error
+
+	glog.V(100).Infof("Adding SCC privileged to the agnhost namespace")
+
+	err = scc.AddPrivilegedSCCtoDefaultSA(egressIPNamespace)
+	if err != nil {
+		return "", fmt.Errorf("failed to add SCC privileged to the agnhost namespace %s: %w",
+			egressIPNamespace, err)
+	}
+
+	glog.V(100).Infof("Removing ServiceAccount")
+
+	err = apiobjectshelper.DeleteServiceAccount(apiClient, proberDeploySAName, egressIPNamespace)
+
+	if err != nil {
+		return "", fmt.Errorf("failed to remove serviceAccount %q from egressIPNamespace %q",
+			proberDeploySAName, egressIPNamespace)
+	}
+
+	glog.V(100).Infof("Creating ServiceAccount")
+
+	err = apiobjectshelper.CreateServiceAccount(apiClient, proberDeploySAName, egressIPNamespace)
+
+	if err != nil {
+		return "", fmt.Errorf("failed to create serviceAccount %q in egressIPNamespace %q",
+			proberDeploySAName, egressIPNamespace)
+	}
+
+	glog.V(100).Infof("Removing Cluster RBAC")
+
+	err = apiobjectshelper.DeleteClusterRBAC(apiClient, proberDeployRBACName)
+
+	if err != nil {
+		return "", fmt.Errorf("failed to delete prober RBAC %q", proberDeployRBACName)
```

Comment:
nit: what does _prober_ stand for?

Reply:
as far as I understand you don't like the 'prober' name. it is kind of the left-over from the first code version, but it was created also to differentiate your methods name with mine (like create/delete Deployment, CM etc.)
Anyway, if you don't like it, so it was removed completely from the egress-ip.go